src/emulator/storage/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const SIGNED_URL_MIN_TTL_SECONDS = 1;
+export const SIGNED_URL_MAX_TTL_SECONDS = 7 * 24 * 60 * 60;
+export const SIGNED_URL_DEFAULT_TTL_SECONDS = 60 * 60;
+export const SECONDS_TO_MS_FACTOR = 1000;
+export const SIGNED_URL_PRIVATE_KEY =
+  "5c39d11c20b96241c42e07218724d6726ee27c3db33eb43256c21f92bb1f52dc";
+export const INVALID_TTL_PARAM = "X-Firebase-Expires: TTL";
```

Comment:
Apologies if I was not clear, can we hardcode all instances of header names into a constant:

```
export const X_FIREBASE_EXPIRES = "X-Firebase-Expires";
export const X_FIREBASE_DATE = "X-Firebase-Date";
export const X_FIREBASE_SIGNATURE = "X-Firebase-Signature";
```

and then replace every string literal "X-Firebase-Expires" with `${X_FIREBASE_EXPIRES}`. The goal is to just have just one place where we define these string.

Reply:
Makes sense. I assumed we just wanted the invalid headers.